interface/src/app/volcano-plot/view.js

```diff
@@ -0,0 +1,34 @@
+angular.module('adage.volcano-plot.view', [
```

Comment:
Another insightful comment. Something didn't feel right about naming the whole directory `volcanoPlot` or `volcano_plot`, so I went with `volcano-plot`. In part, I chose this because that's how Angular modules themselves are organized. Inside `interface/vendor/`, you can find `angular-animate`, `angular-bootstrap`, `angular-mocks`, etc. I followed that pattern.

I'm flexible, however. If you don't like it, I can switch it to `volcanoPlot`. But not `volcano_plot`. Underscores bother me. Hmm, maybe that's the real reason I prefer `volcano-plot` over the others: both `volcanoPlot` and `volcano_plot` require me to make use of the shift key to type them. Darned shift key.